source/Octopus.Cli/Util/FeatureDetectionExtensions.cs

```diff
@@ -7,22 +7,12 @@ public static class FeatureDetectionExtensions
     {
         public static bool SupportsChannels(this IOctopusAsyncRepository repository)
```

Comment:
I'm not sure if we should remove this.

>IIRC, it will return false if the default space is off and SpaceId is not provided, however does that false really mean channels are not supported?

Good point. Maybe we should update this check to see if `"SpaceHome"` exists on the root resource, as we should know that channels exist post-spaces.

Reply:
We could do the same thing for tenants, since that will have the same problem

Reply:
Sounds good, I will add the "SpaceHome" check to both channels and tenants